lib/shopt.sh

```diff
@@ -27,7 +27,15 @@ shopt -s nocaseglob;
 ## SMARTER TAB-COMPLETION (Readline bindings) ##
 
 # Perform file completion in a case insensitive fashion
-bind "set completion-ignore-case on"
+# Note: CASE_SENSITIVE is the compatibility name
+if [[ ${OMB_CASE_SENSITIVE:-${CASE_SENSITIVE:-}} == false ]]; then
+	bind "set completion-ignore-case on"
+elif [[ ${OMB_CASE_SENSITIVE:-${CASE_SENSITIVE:-}} == true ]]; then
+	bind "set completion-ignore-case off"
+else
+	# By default, case sensitivity is disabled.
+	bind "set completion-ignore-case on"
+fi
```

Comment:
These three branches seem still redundant, but this seems to be addressed in #399 by @jjmcdn.

Also, I'd like to keep the original state when `[OMB_]CASE_SENSITIVE` is unspecified. The users may have a separate setting in e.g. `~/.inputrc`, which we wouldn't like to override.

Reply:
> Also, I'd like to keep the original state when `[OMB_]CASE_SENSITIVE` is unspecified.

Hmm, but OMB seems to have set `completion-ignore-case` always on so far, so it is reasonable to keep that behavior. Let me mark this conservation *Resolved*.